core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -154,9 +154,9 @@ describe('Voiceover upload features', function() {
   afterEach(async function() {
     await general.checkForConsoleErrors([
       'Failed to load resource: the server responded with a status of 400' +
-      '(Bad Request)', {status_code: 400,
+      '(Bad Request)', { status_code: 400,
```

Comment:
Based on [DubeySandeep's comment on this PR](https://github.com/oppia/oppia/pull/12627#issuecomment-842738111), I don't think we're expecting this kind of change. ditto for the rest of the e2e test files